ts/src/types.ts

```diff
@@ -6,12 +6,25 @@ import {
   AppWebsocket,
   CallZomeRequest,
   CapSecret,
+  DnaProperties,
   DnaSource,
+  DnaBundle,
+  HoloHash,
   InstalledCell,
   MembraneProof,
   RoleId,
+  RegisterDnaRequest,
 } from "@holochain/client";
 
+export type RegisterDnaReqOpts = Omit<
```

Comment:
This is still an awkward part of TypeScript (or there is a way how to prevent this that I don't know about)

Reply:
:100: Agreed!